src/stormpath.social-login.facebook.js

```diff
@@ -0,0 +1,67 @@
+(function() {
+  'use strict';
```

Comment:
Since files are concatenated and minified in production, it's best to specify the strict mode for this scope only: <http://yuiblog.com/blog/2010/12/14/strict-mode-is-coming-to-town/>